pkg/cli/cmd/error.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+)
+
+type Error struct {
+	UserMessage string
+	LogMessage  string
+}
+
+func LogError(err *Error, checkLogMessage string) {
```

Comment:
Why pass in the check message instead of making the constant here? If this is the one responsible for actually printing the messages to the screen, IMO it feels more natural here instead of passing it in each time.

Reply:
Because it's not a constant. The different subcommands pass different `checkLogMessage`.